arch/arm/src/Makefile

```diff
@@ -165,7 +166,10 @@ define LINK_ALLSYMS
 	$(Q) $(call DELFILE, allsyms.tmp allsyms$(OBJEXT))
 endef
 
-nuttx$(EXEEXT): $(HEAD_OBJ) board$(DELIM)libboard$(LIBEXT) $(ARCHSCRIPT)
+$(addsuffix .tmp,$(ARCHSCRIPT)): $(ARCHSCRIPT)
```

Comment:
```suggestion
$(ARCHSCRIPT).tmp: $(ARCHSCRIPT)
```

Reply:
$(ARCHSCRIPT) will contain multiple targets

Reply:
If so, we need use % rule here like .c->.o

Reply:
ARCHSCRIPT do not have uniform naming rules and may exist in different directories, so the % rules may not cover for all cases